package.json

```diff
@@ -201,6 +201,7 @@
     "react-router-redux": "4.0.8",
     "react-scrollspy": "3.3.3",
     "react-waypoint": "8.0.0",
+    "react-zendesk": "^0.1.10",
```

Comment:
Looks like the amount of work to reactify the Zendesk JS snippet was [minimum](https://github.com/B3nnyL/react-zendesk/blob/03a059046a362f41bf30569897436558b77eb720/src/index.js). I wonder if we should just "own our dependencies" by implementing it in Force directly. We will benefit from faster iteration (such as supporting `defer`) and adding tests (😱 ). @damassi Curious if we have recommendation in this case.

Reply:
Both approaches could work! I don't have a preference.

Reply:
We could also use this https://github.com/ds300/patch-package

Reply:
This was merged. Should I update the PR to use the latest version?

Reply:
Yep let's update to the latest version!